controller/scheduler/fleet.py

```diff
@@ -377,6 +376,7 @@ def attach(self, name):
 
 CONTAINER_TEMPLATE = [
     {"section": "Unit", "name": "Description", "value": "{name}"},
+    {"section": "Service", "name": "EnvironmentFile", "value": "/etc/environment"},
```

Comment:
I think it would be ideal if you demonstrated how ETCDCTL_PEERS would be set up via cloud config in this PR. AFAICT users would have to manage this outside of cloud-config since it's a cluster-wide setup.